mne/_freesurfer.py

```diff
@@ -504,7 +524,23 @@ def read_lta(fname, verbose=None):
     _validate_type(fname, ('path-like', None), 'fname')
     _check_fname(fname, 'read', must_exist=True)
     with open(fname, 'r') as fid:
-        affine = np.loadtxt(fid.readlines()[5:9])
+        lines = fid.readlines()
+    # 0 is linear vox2vox, 1 is linear ras2ras
+    trans_type = int(lines[0].split('=')[1].strip()[0])
+    assert trans_type in (0, 1)
+    affine = np.loadtxt(lines[5:9])
+    if trans_type == 1:
+        return affine
+
+    src_affine = _get_affine_from_lta_info(lines[12:18])
+    dst_affine = _get_affine_from_lta_info(lines[21:27])
+
+    # don't compute if src and dst are already identical
+    if np.allclose(src_affine, dst_affine):
```

Comment:
just checking: are the default tolerances sensible here?

Reply:
They should be exactly the same since they are both stored in the lta file the same but I think this just gives a little grace for machine imprecision. We can change it to test that they are exactly equal if you'd prefer...

Reply:
ok. I don't want to cause ourselves headaches down the road by being too stringent.  But `np.allclose` is a lot more forgiving than is needed for machine imprecision, and I'm in the habit of always asking when I see `allclose` used with the defaults. This case in particular, the last column of the affine is often 2 or 3 orders of magnitude bigger than the rest, which is also a reason to pause and think about whether `1e-5 + 1e-8 * data` is a reasonable amout to allow these to differ by and still be considered "equal".

Reply:
However you want to do it is fine, for practical purposes, it will yield the same result up to the precision of the image resolution anyway.